services/queue/src/dependencyresolver.js

```diff
@@ -43,67 +44,65 @@ class DependencyResolver {
     this._pollingDelay = options.pollingDelay;
     this._parallelism = options.parallelism;
 
-    // Promise that polling is done
-    this._done = null;
-    // Boolean that polling should stop
-    this._stopping = false;
+    // do iteration
+    this.iterator = new Iterate({
+      maxFailures: 10,
+      waitTime: this._pollingDelay,
+      watchDog: this._pollingDelay,
```

Comment:
I think that this wants to be larger than `maxIterationTime`, since we don't call `watchDog.touch()`.